dsgrid/query/peak_load_report.py

```diff
@@ -0,0 +1,56 @@
+import logging
```

Comment:
If this file were named report_peak_load.py, that would start to group reports in a similar spot.